src/app/forms/templates/trl/trl-tyres.template.ts

```diff
@@ -0,0 +1,100 @@
+  children: [
+    {
+      name: 'tyreUseCode',
+      label: 'Tyre use code',
+      value: '',
+      type: FormNodeTypes.CONTROL,
+      validators: [
+        { name: ValidatorNames.MaxLength, args: 2 },
+        { name: ValidatorNames.Min, args: 0 }
+      ]
+    },
+    {
+      name: 'axles',
+      value: '',
+      type: FormNodeTypes.ARRAY,
+      children: [
+        {
+          name: '0',
+          label: 'Axle',
+          value: '',
+          type: FormNodeTypes.GROUP,
+          children: [
+            {
+              name: 'axleNumber',
+              label: 'Axle Number',
+              type: FormNodeTypes.CONTROL,
+              validators: [
+                { name: ValidatorNames.Numeric },
+                { name: ValidatorNames.Max, args: 99999 },
+                { name: ValidatorNames.Min, args: 0 },
+                { name: ValidatorNames.Required }
+              ]
+            },
+            {
+              name: 'tyres',
+              label: 'Tyres',
+              value: '',
+              type: FormNodeTypes.GROUP,
+              children: [
+                {
+                  name: 'tyreCode',
+                  label: 'Tyre Code',
+                  value: '',
+                  type: FormNodeTypes.CONTROL,
+                  validators: [
+                    { name: ValidatorNames.Numeric },
+                    { name: ValidatorNames.Max, args: 99999 },
+                    { name: ValidatorNames.Min, args: 0 },
+                    { name: ValidatorNames.Required }
+                  ]
+                },
+                {
+                  name: 'tyreSize',
+                  label: 'Tyre Size',
+                  value: '',
+                  type: FormNodeTypes.CONTROL,
+                  disabled: true,
+                  validators: [{ name: ValidatorNames.MaxLength, args: 12 }, { name: ValidatorNames.Min, args: 0 }, { name: ValidatorNames.Required }]
+                },
+                {
+                  name: 'plyRating',
+                  label: 'Ply Rating',
+                  value: '',
+                  type: FormNodeTypes.CONTROL,
+                  disabled: true,
+                  validators: [
+                    { name: ValidatorNames.MaxLength, args: 2 },
+                    { name: ValidatorNames.Min, args: 0 }
+                  ]
+                },
+                {
+                  name: 'fitmentCode',
+                  label: 'Fitment code',
+                  value: '',
+                  type: FormNodeTypes.CONTROL,
+                  validators: [{ name: ValidatorNames.Required }]
+                },
+                {
+                  name: 'dataTrAxles',
+                  label: 'Load index',
+                  value: '',
+                  type: FormNodeTypes.CONTROL,
+                  disabled: true,
+                  validators: [{ name: ValidatorNames.Numeric }, { name: ValidatorNames.Max, args: 999 }, { name: ValidatorNames.Min, args: 0 }]
```

Comment:
Not editable?

Reply:
Yeah, they are filled in by the ref data!

Reply:
How would the validators handle if the ref data came back invalid? If the user has no control of what's in that field, does it make sense to have validators on it?